### PR TITLE
 Docs review

### PR DESCRIPTION
**Pull request by detonivictor**
## Summary

- added activities and interests on user.md
- added a troubleshooting about plugin version conflict
- added cartModified on events.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,14 @@ application's public ID.
 <script src="https://cdn.croct.io/js/v1/lib/plug.js?appId=<APP_ID>"></script>
 ```
 
+### Function call
```
Comment (marcospassos) [inline README.md:134]: You need dedicated sections for each installation method as they are not the same.

Suggestion:

### NPM

The recommended way to install the SDK is via [NPM](https://npmjs.com). It pairs nicely with module bundlers such as Webpack or Browserify and includes Typescript typings.

In most cases, it should be as simple as running the following in your project:

```sh
npm install @croct/plug
```

Then, call [`croct.plug`](docs/plug.md#plug) passing the App ID to  initialize the SDK:

```js
import croct from '@croct/plug';

croct.plug({appId: '<APP_ID>'});
```
### Script Tag

To install the SDK as a script tag, add the following line to the `<head>` tag of your site on any pages you plan to use the SDK to personalize or track events:

```html
<script src="https://cdn.croct.io/js/v1/lib/plug.js?appId=<APP_ID>"></script>
<script>croct.plug();</script>
```

You should replace the `<APP_ID>` placeholder with the respective App ID.  For more information about the available options, see [`croct.plug`](docs/plug.md#plug).

File: docs/events.md [hunk 2]
```diff
@@ -52,6 +53,7 @@ There are several event types that you can record within the customer journey:
 | [`pageLoaded`](#pageloaded)                             | Web           | Yes           | Records a page load.
 | [`productViewed`](#productviewed)                       | E-commerce    | No            | Records a product view.
 | [`cartViewed`](#cartviewed)                             | E-commerce    | No            | Records a cart view.
+| [`cartModified`](#cartmodified)                         | E-commerce    | No            | Records a cart modify.
```
Comment (marcospassos) [inline docs/events.md:56]: `Records a shopping cart change.` Please, also update `Records a cart view.` to `Records a shopping cart view` to keep consistency.

File: docs/events.md [hunk 3]
```diff
@@ -429,6 +431,148 @@ croct.track('cartViewed', {
 ```
 </details>
 
+### cartModified
+
+This event records the user modified the shopping cart.
```
Comment (marcospassos) [inline docs/events.md:436]: You should track this event when a user modifies the shopping cart page, either by adding, removing, or updating items.

File: docs/events.md [hunk 4]
```diff
@@ -429,6 +431,148 @@ croct.track('cartViewed', {
...
+
+You should track this event when a user modifies the shopping cart page or summary.
+
+#### Properties
+
+This event supports the following properties:
+
+| Property                              | Type     | Required | Constraints                       | Description                                                                                                                                                                                                                                                                                                                                                 
+|---------------------------------------|----------|----------|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```
Comment (marcospassos) [inline docs/events.md:445]: The formating/indentation is completely broken. Please refer to other sections for examples of how you should format this table.

File: docs/user.md [hunk 5]
```diff
@@ -46,28 +46,30 @@ This method creates a patch to apply changes to the user's profile.
...
+| `address.district`   | `String` | Between 1 and 100 characters long                 | The address' district.
+| `address.city`       | `String` | Between 1 and 100 characters long                 | The address' city.
+| `address.region`     | `String` | Between 1 and 100 characters long                 | The address' region.
+| `address.country`    | `String` | Between 1 and 100 characters long                 | The address' country.
+| `address.postalCode` | `String` | Between 1 and 20 characters long                  | The address' postal code.
+| `avatar`             | `String` | Well-formed URL                                   | The personal avatar URL.
+| `company`            | `String` | Between 1 and 200 characters long                 | The company's name.
+| `companyUrl`         | `String` | Well-formed URL                                   | The company's website URL.
+| `jobTitle`           | `String` | Between 1 and 50 characters long                  | The job title.
+| `interests`          | `array`  | Up to 30 strings between 1 and 30 characters long | The user's interests
```
Comment (marcospassos) [inline docs/user.md:70]: `The demonstrated interests.`
`The performed activities.`

**Review by marcospassos — CHANGES_REQUESTED** [review]

**Review by marcospassos — CHANGES_REQUESTED** [review]
Please fix the indicated issues and request a new review.

File: docs/troubleshooting.md [hunk 6]
```diff
@@ -46,4 +47,40 @@ You can temporarily collect information to let the session unification process a
...
+```js
+croct.plug({
+    debug: true,
+    plugins: {
+        googleAnalytics: true,
+    }
+});
+```
+
+2. Open your browser and check if there is an error like “Plugin ‘pluginName’ is not registered” in the console
```
Comment (marcospassos) [inline docs/troubleshooting.md:82]: Using double quotes `"`

File: docs/troubleshooting.md [hunk 7]
```diff
@@ -46,4 +47,40 @@ You can temporarily collect information to let the session unification process a
...
+        // Enable the plugin
+        googleAnalytics: true,
+    }
+});
+```
+
+If the problem persists, check if the plugin has been found and loaded. To do this, follow the steps below:
+
+1. Enable the debug mode
+
```
Comment (marcospassos) [inline docs/troubleshooting.md:72]: remove empty line if possible

File: docs/troubleshooting.md [hunk 8]
```diff
@@ -46,4 +47,40 @@ You can temporarily collect information to let the session unification process a
...
+```js
+croct.plug({
+    debug: true,
+    plugins: {
+        googleAnalytics: true,
+    }
+});
+```
+
```
Comment (marcospassos) [inline docs/troubleshooting.md:81]: remove empty line if possible

File: README.md [hunk 9]
```diff
@@ -121,16 +121,25 @@ In most cases, it should be as simple as running the following in your project:
...
-to use the SDK to personalize or track events. Next, replace the `<APP_ID>` placeholder with the respective 
-application's public ID.
+To install the SDK as a script tag, add the following line to the `<head>` tag of your site on any pages you plan to use the SDK to personalize or track events:
 
 ```html
 <script src="https://cdn.croct.io/js/v1/lib/plug.js?appId=<APP_ID>"></script>
+<script>croct.plug();</script>
 ```
 
+You should replace the `<APP_ID>` placeholder with the respective App ID. For more information about the available options, see [croct.plug](docs/plug.md#plug).
```
Comment (marcospassos) [inline README.md:141]: `croct.plug`

File: docs/events.md (hunk 4 above)
Comment (detonivictor) [reply docs/events.md:445]: Have you verified if there is really an indentation problem?

Comment (marcospassos) [reply docs/events.md:445]: Never mind, that's an issue with the Github UI

File: docs/troubleshooting.md [hunk 10]
```diff
@@ -46,4 +47,38 @@ You can temporarily collect information to let the session unification process a
...
+1. Enable the debug mode
+```js
+croct.plug({
+    debug: true,
+    plugins: {
+        googleAnalytics: true,
+    }
+});
+```
+2. Open your browser and check if there is an error like "Plugin ‘pluginName’ is not registered" in the console
```
Comment (marcospassos) [inline docs/troubleshooting.md:80]: Wrong quotes

Comment (marcospassos) [reply docs/troubleshooting.md:80]: 2. Open your browser and check if there is an error like _"Plugin 'pluginName' is not registered"_ in the console

Comment (marcospassos) [reply docs/troubleshooting.md:80]: keep in italic, pls

File: README.md [hunk 11]
```diff
@@ -121,16 +121,25 @@ In most cases, it should be as simple as running the following in your project:
 npm install @croct/plug
 ```
 
+Then, call `croct.plug` passing the App ID to initialize the SDK:
```
Comment (marcospassos) [inline README.md:124]: Please add a link to the method's doc

**Review by marcospassos — APPROVED** [review]

**Comment by marcospassos** [comment]
Thank you for the collaboration, @detonivictor!

**Pull request merged**